fe/src/project/Seller.Portal/areas/Clients/pages/ClientPage/ClientPage.js

```diff
@@ -18,7 +18,7 @@ function ClientPage(props) {
         <div>
           <Header {...props.header}></Header>
           <MenuTiles {...props.menuTiles} />
-          <ClientForm {...props.clientForm} />
+          <ClientForm {...props.clientForm} locale={props.locale} />
```

Comment:
@SebaKas21 , when we use TextField instead of ReactPhoneInput it can be safely removed. I'd give flexibility here instead of restricting it too much.